modules/core/src/test/java/org/apache/ignite/internal/processors/datastreamer/MultipleDataStreamersOnClientTest.java

```diff
@@ -1,253 +1,253 @@
-/*

```

Comment:
could you please highlight what had changed here?

Reply:
Contents have differences only in line separators